### PR TITLE
[CARBONDATA-872] Fix comment issues of integration/presto for easier reading

### PR DESCRIPTION
**Pull request by ffpeng90**
Fix comment issues of integration/presto for easier reading

### CODE REVIEW

**Comment by chenliang613** [comment]
@ffpeng90  can you optimize other code also, thanks.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1446/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1447/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1464/

File: integration/presto/src/checkstyle/checks.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
```
Comment (chenliang613) [inline integration/presto/src/checkstyle/checks.xml]: Please remove this file.

File: integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataRecordCursor.java [hunk 2]
```diff
@@ -78,6 +78,10 @@ public CarbondataRecordCursor(CarbonReadSupport<Object[]> readSupport,
     return columnHandles.get(field).getColumnType();
   }
 
+  /**
+   * get next Row/Page
+   * @return
+     */
```
Comment (chenliang613) [inline integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataRecordCursor.java]: please change format the comment as below : 
/**
 * get next Row/Page
 * @return
 */

File: integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataSplitManager.java [hunk 3]
```diff
@@ -106,6 +110,12 @@ public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHand
     return constraintBuilder.build();
   }
 
+  /**
+   * Convert presto-TupleDomain predication into Carbon scan express condition
+   * @param originalConstraint  presto-TupleDomain
+   * @param carbonTable
+   * @return
+     */
```
Comment (chenliang613) [inline integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataSplitManager.java]: please change the comment format.

File: integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataSplitManager.java [hunk 4]
```diff
@@ -249,6 +250,11 @@ public Expression parseFilterExpression(TupleDomain<ColumnHandle> originalConstr
     return finalFilters;
   }
 
+  /**
+   * Convert presto spi Type into Carbondata Type
+   * @param colType
+   * @return
+     */
```
Comment (chenliang613) [inline integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataSplitManager.java]: please change the comment format

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1482/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1488/

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**